content/blog/Galaxy_Climate_Project_Introduction_Blog_1/index.md

```diff
@@ -0,0 +1,101 @@
+
+
+
+
+# What are the existing CLIMATE TOOLS available in the GALAXY TOOLSHED?
+
+The Galaxy ToolShed houses more than 2,000 tools, out of which several tools are helpful for Climate Analysis.  Some of the relevant tags to find under the Galaxy ToolShed are: “Climate Analysis”, “Ecology”, etc. You can follow the list given below for more clarity. 
+
+
+## CLIMATE ANALYSIS TOOLS
+| Climate Analysis Tools |      Synopsis    |
+|----------|:-------------:|
+| [c3s](https://toolshed.g2.bx.psu.edu/repository/browse_repository?id=66db1d2dd5ec1fab) |    Copernicus Climate Change Service (C3S)   |  
+| [cads](https://toolshed.g2.bx.psu.edu/repository/browse_repositories_in_category?sort=name&operation=view_or_manage_repository&id=58c4b9b8233320fc) |  Copernicus Atmosphere Data Store (ADS)   |
+|[cdo_info](https://toolshed.g2.bx.psu.edu/repository/browse_repositories_in_category?sort=name&operation=view_or_manage_repository&id=c8c1cfa654cb096d) | Wrapper for cdo tool: CDO get info. |
+| [cdo_operations](https://toolshed.g2.bx.psu.edu/repository/browse_repositories_in_category?sort=name&operation=view_or_manage_repository&id=8a2394bf5c1816a5) |  Wrapper for cdo tool: CDO Operations. |
+| [cds_essential_variability](https://toolshed.g2.bx.psu.edu/repository/browse_repositories_in_category?sort=name&operation=view_or_manage_repository&id=0d422cfb5d8fef89)|   Get Copernicus Essential Climate Variables for assessing climate variability   |
+|[cesm](https://toolshed.g2.bx.psu.edu/repository/browse_repositories_in_category?sort=name&operation=view_or_manage_repository&id=7aa3cab2c60dddc0) | Community Earth System Model (CESM) |
+| [climate_stripes](https://toolshed.g2.bx.psu.edu/repository/browse_repositories_in_category?sort=name&operation=view_or_manage_repository&id=51707a18875902ba) |  Create climate stripes from a tabular input file |
+| [ctsm_fates](https://toolshed.g2.bx.psu.edu/repository/browse_repositories_in_category?sort=name&operation=view_or_manage_repository&id=f39a869bd7709275) |    EMERALD version of the Functionally Assembled Terrestrial Ecosystem Simulator (FATES) with Community Terrestrial Systems Model as host model  |
+| [eodie](https://toolshed.g2.bx.psu.edu/repository/browse_repositories_in_category?sort=name&operation=view_or_manage_repository&id=ddd8dba334f1d765) |  Earth Observation Data Information Extractor |
+| [psy_maps]() | Visualization of regular geographical data on a map with psyplot |
+|[shift_longitudes](https://toolshed.g2.bx.psu.edu/repository/browse_repositories_in_category?sort=name&operation=view_or_manage_repository&id=6f6f2aca88e226de) | Shift longitudes ranging from 0. and 360 degrees to -180. and 180. degrees |

```

Comment:
if these are degrees, wouldn't it be 180º?

Reply:
done